crates/weaver_semconv/src/body.rs

```diff
@@ -0,0 +1,244 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(rustdoc::invalid_html_tags)]
+
+//! Body Field specification.
+
+use serde::{Deserialize, Serialize};
+
+use crate::attribute::{
+    AttributeType, BasicRequirementLevelSpec, Examples, RequirementLevel, ValueSpec,
+};
+use crate::stability::Stability;
+
+/// A body specification
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(deny_unknown_fields)]
+#[serde(untagged)]
+#[serde(rename_all = "snake_case")]
+pub enum BodySpec {
+    /// The collection of body fields associated with a body definition
+    Fields {
+        /// The collection of body fields associated with a body definition
+        #[serde(skip_serializing_if = "Vec::is_empty")]
+        fields: Vec<BodyFieldSpec>,
+    },
+    /// The body field value.
+    Value {
```

Comment:
What's the difference between value and fields?

Reply:
https://github.com/open-telemetry/semantic-conventions/issues/755

Reply:
Commented my thoughts here.  Seeing this in action - I'm not a fan.